services/collaboration/pkg/config/defaults/defaultconfig.go

```diff
@@ -0,0 +1,83 @@
+package defaults
+
+import (
+	"github.com/dchest/uniuri"
+	"github.com/owncloud/ocis/v2/services/collaboration/pkg/config"
+)
+
+// FullDefaultConfig returns a fully initialized default configuration
+func FullDefaultConfig() *config.Config {
+	cfg := DefaultConfig()
+	EnsureDefaults(cfg)
+	Sanitize(cfg)
+	return cfg
+}
+
+// DefaultConfig returns a basic default configuration
+func DefaultConfig() *config.Config {
+	return &config.Config{
+		Service: config.Service{
+			Name: "collaboration",
+		},
+		App: config.App{
+			Name:        "WOPI app",
+			Description: "Open office documents with a WOPI app",
+			Icon:        "image-edit",
+			LockName:    "com.github.owncloud.collaboration",
+		},
+		Secret: uniuri.NewLen(32),
+		GRPC: config.GRPC{
+			Addr:      "127.0.0.1:56778",
+			Namespace: "com.owncloud.collaboration",
+		},
+		HTTP: config.HTTP{
+			Addr:      "127.0.0.1:6789",
```

Comment:
How does this fit in here https://owncloud.dev/services/general-info/port-ranges/ ?